packages/@aws-cdk/aws-codepipeline-actions/lib/codecommit/source-action.ts

```diff
@@ -54,6 +77,13 @@ export interface CodeCommitSourceActionProps extends codepipeline.CommonAwsActio
    * The CodeCommit repository.
    */
   readonly repository: codecommit.IRepository;
+
+  /**
+   * The variables that this action will emit.
+   *
+   * @default - no variables will be emitted from this action
+   */
+  readonly variables?: CodeCommitSourceVariables;
```

Comment:
I think this should just be a boolean

Reply:
See [my reply above](https://github.com/aws/aws-cdk/pull/5604#discussion_r362601456).